SonarQube.TeamBuild.Integration/Resources.resx

```diff
@@ -158,6 +158,9 @@ Check that the downloaded code coverage file ({0}) is valid by opening it in Vis
   <data name="PROC_ERROR_FailedToDownloadReport" xml:space="preserve">
     <value>Failed to download the code coverage report.</value>
   </data>
+  <data name="PROC_ERROR_FailedToDownloadReportReason" xml:space="preserve">
+    <value>"Failed to download the code coverage report. The HTTP status code was {0} and the reason \"{1}\""</value>
```

Comment:
Ok, I've added the URL to the error message so that it is clearer. Note that there already exists an INFO message containing the URL.